src/Microsoft.AspNetCore.Server.IntegrationTesting/Common/DeploymentParameters.cs

```diff
@@ -13,6 +14,35 @@ namespace Microsoft.AspNetCore.Server.IntegrationTesting
     /// </summary>
     public class DeploymentParameters
     {
+        public DeploymentParameters()
```

Comment:
We are having to many entities now, why do we need both this and variations with the same information?

Reply:
The TestVariant has pretty strict requirements around serialization. DeploymentParameters has a lot more state including things like `IDictionary<string, string>` and `Action<DeploymentParameters>`. I'll see if I can collapse DeploymentParameters into the ApplicationDeployer base class.

Reply:
Consolidating DeploymentParameters is going to cause way too much churn for this pass with no functional benifit. MVC has a whole test infrastructure built around it.